variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -220,16 +227,23 @@ def process_bam_file(bam_file_path, output_file, out_failed_file, new_genome, fi
             if pass_basic_filtering(primary_group, secondary_group, primary_to_supplementary, counter, filter_align_with_secondary):
                 left_read, right_read = _order_reads(primary_group, primary_to_supplementary)
                 if pass_aligned_filtering(left_read, right_read, counter):
-                    counter['Remapped'] += 1
-                    varpos, new_ref, new_alts, ops = calculate_new_variant_definition(left_read, right_read, fasta)
-                    info = left_read.query_name.split('|')
-                    if info[7] != '.':
-                        info[7] += ';'.join(ops)
+                    varpos, new_ref, new_alts, ops, low_confidence = calculate_new_variant_definition(left_read, right_read, fasta)
```

Comment:
```suggestion
                    varpos, new_ref, new_alts, ops, low_confidence = \
                        calculate_new_variant_definition(left_read, right_read, fasta)
```

(Line over 120 columns)